### PR TITLE
Change CLA missing to tell people to check themselves

### PR DESCRIPTION
**Pull request by CoolCat467**
CLA not signed tag is not automatically removed after records are updated, so add a line to tell people to check their username with the herokuapp link after they have signed the CLA. The better solution would be to automatically update CLA not signed tag after CLA signed record is in place, but that would mean a task has to be run automatically and that might be hard or cause security flaws or something. Easier to tell people to check themselves after they sign it.

### CODE REVIEW

File: ni/github.py [hunk 1]
```diff
@@ -50,7 +50,8 @@
 to rectify this issue.
 
 If you have recently signed the CLA, please wait at least one business day
-before our records are updated.
+before our records are updated. After our records are updated, please
+[check your username](https://check-python-cla.herokuapp.com/)
```
Comment (hugovk) [inline ni/github.py:54]:
```suggestion
[check your username](https://check-python-cla.herokuapp.com/).
```

**Review by hugovk — COMMENTED** [review]
It's been a while since I signed it, do people get some kind of notification when the records have been updated?

That is, do they have a prompt to go and check their username?

**Comment by hugovk** [comment]
Let's close because this project has been replaced and will be archived: https://github.com/python/the-knights-who-say-ni/issues/320. Thanks anyway for the PR!

**Comment by CoolCat467** [comment]
Oh well.

**Pull request closed without merging**